RNAndroidPlayground/app/src/main/java/com/kiwi/rnandroidplayground/HotelActivityFragment.kt

```diff
@@ -0,0 +1,37 @@
+package com.kiwi.rnandroidplayground
+
+import android.app.Activity
+import android.os.Bundle
+
+import com.facebook.react.modules.core.DefaultHardwareBackBtnHandler
+
+class HotelActivityFragment : Activity(), DefaultHardwareBackBtnHandler {
+
+  companion object {
+    fun getViewModelClass(): Class<HotelActivityFragment> =
```

Comment:
I do not get why is this method calleed getViewModelClass when it is actually returning view class?